### ISSUE TITLE
Change references to "jade" to "pug"

### ISSUE THREAD

**Issue by crandmck** [expressjs/expressjs.com#609]
Per https://github.com/expressjs/express/issues/2952, need to update http://expressjs.com/en/guide/using-template-engines.html (and search for any other instances of "jade").

**Comment by Knighton910** [comment]
I'm doing a few updates to express guides, i can handle this, but it will be several days or so, do you wish to take care of this, or shall i get to it?

**Comment by crandmck** [comment]
@lordKnighton Please do!  I've been meaning to get to this, but just haven't had a chance.  Thanks!!

**Comment by Knighton910** [comment]
no worries

**Comment by crandmck** [comment]
@lordKnighton Are you going to take another crack at this?  If not, then I'll try to do it.  No worries either way, just let me know.

**Comment by Knighton910** [comment]
go for it, i got swapped. I can do it next week, if you prefer me to do it.

**Comment by crandmck** [comment]
OK, PR updates the English docs, and I opened #618 to track task to update translated docs.  Also added #617 for when the generator is updated.

@lordKnighton (or anyone) pls take a quick look at the PR ^

**Comment by Knighton910** [comment]
👍

**Comment by crandmck** [comment]
Thanks!  
We'll address non-en docs as time permits, and generator docs when the generator is updated.

**Comment by Knighton910** [comment]
+1

**Comment by martinjoiner** [comment]
Hey guys.  I am a firs-timer here on this repo. I would like to contribute by replacing references to Jade with Pug in the "Express Application Generator" page http://expressjs.com/en/starter/generator.html  I recently got confused when followed this guide to bootstrap a project and then tried to find Jade documentation. Is it OK to go ahead and fork, edit and create a pull request?

**Comment by crandmck** [comment]
Hi @martinjoiner thanks for offering to contribute.

This seems reasonable, and IIRC, the only reason we haven't made this change was because Pug was still in beta.  Although npm still says version if 2.0.0-beta11, it's replaced Jade now for quite a while, so I'm not sure if this is still an issue.

Even though the generator still supports Jade, I agree we should start nudging people toward Pug, since it's being maintained instead of Jade now (due to the naming issue).

However, I'll defer to @dougwilson on this.... Doug, should we wait for Pug non-beta release to make this sort of change?

**Comment by dougwilson** [comment]
I don't see any reason any longer not to just update that generator page to use `--view=pug` if that reduces confusion 😄

**Comment by martinjoiner** [comment]
I have forked, edited and created a pull request (https://github.com/expressjs/expressjs.com/pull/785). My initial motivation for making this change was to save confusion for English-speaking new adopters. However, as I worked on the different language versions I realise how the real win here is for the non-English-speaking users who might find it harder than I did to learn about the forced renaming.
